### ISSUE TITLE
AVT 1 - NumberInput DAP violation for ID references

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4198]
## Enviroment
macOS 10.14.6
Chrome Version 77.0.3865.90
VoiceOver
DAP 1.8.0.0

## Description
Run DAP on the NumberInput Component and the following violation displays:
<img width="1064" alt="DAP violation screen shot" src="https://user-images.githubusercontent.com/21676914/66136388-006b6d80-e5c1-11e9-8fea-4ed92997b145.png">
